pkg/services/auth/auth_token.go

```diff
@@ -60,7 +61,7 @@ func (s *UserAuthTokenService) ActiveTokenCount(ctx context.Context) (int64, err
 	return count, err
 }
 
-func (s *UserAuthTokenService) CreateToken(ctx context.Context, userId int64, clientIP net.IP, userAgent string) (*models.UserToken, error) {
+func (s *UserAuthTokenService) CreateToken(c *models.ReqContext, user *models.User) (*models.UserToken, error) {
```

Comment:
I understand you want more context available for auditing purpose. Is the idea to extract additional request headers and so on in enterprise from models.ReqContext or what's the purpose of this?

Reply:
Besides the client IP address and the user agent, I needed the request URI information. Leo helped me find a solution where I only add the information I need and not the whole `ReqContext` object.